framework/tests/value/VarArgRe.java

```diff
@@ -0,0 +1,24 @@
+import org.checkerframework.common.value.qual.IntVal;
+import testlib.lib.VarArgMethods;
+
+public class VarArgRe {
+    // VarArgMethods is declarded in framework/tests/src/testlib/lib.
+    // All the methods return the length of the vararg.
+    public void use() {
```

Comment:
Confusing method names.  I would call them `use0`, `use1`, and `use2`, and put them in order.